jwst/ami/NRM_Model.py

```diff
@@ -43,73 +52,99 @@ def __init__(self, mask=None, holeshape="circ",
            shape of apertures
 
         pixscale: float
-           initial estimate of pixel scale in radians.??
+           initial estimate of pixel scale in radians
 
         rotate: float
-           initial estimate of rotation in radians.??
+           initial estimate of rotation in radians
 
         over: integer
            oversampling factor
 
+        flip: Boolean, default=False
+            change sign of 2nd coordinate of holes
+
+        pixweight: 2D float array, default is None
+            weighting array
+
         scallist: float 1D array
            candidate relative pixel scales
 
         rotlist_deg: float 1D array
             Search window for rotation fine-tuning, in degrees
 
+        phi: float 1D array
+            distance of fringe from hole center in units of waves
         """
         self.holeshape = holeshape
         self.pixel = pixscale
-
-        if over:
-            self.over = over
-        else:
-            self.over = 1
-
+        self.over = over
         self.maskname = mask
-        self.ctrs = NRM_consts.ctrs # this is in the 'flip' code block
-        self.ctrs = leastsqnrm.flip(self.ctrs)
-
-        self.d = 0.80  # dg - what is this ?
-        self.D = 6.5   # dg - what is this ?
-        self.N = len(self.ctrs)  # dg - what is this ?
+        self.pixweight = pixweight
+
+        if mask.lower() == 'jwst':
+            self.ctrs = np.array( [[ 0.00000000, -2.640000],
+                                   [-2.2863100, 0.0000000],
+                                   [ 2.2863100 , -1.3200001],
+                                   [-2.2863100 ,  1.3200001],
+                                   [-1.1431500 ,  1.9800000],
+                                   [ 2.2863100 ,  1.3200001],
+                                   [ 1.1431500 ,  1.9800000]] )
+            self.d = 0.80
+            self.D = 6.5
+        else:
+            try:
+                log.debug('mask.ctrs:%s', mask.ctrs)
+            except AttributeError:
+                raise AttributeError("mask must be either 'jwst' \
+                                                      or NRM_mask_geometry object")
 
         log.info('NRM_Model: ctrs flipped in init for CV1, CV2')
```

Comment:
This should probably be a debug level message, rather than info. Don't need to see it during every normal processing run.